SygicMapsKit/Classes/NavigationModule/SYMKNavigationViewController.swift

```diff
@@ -195,15 +198,18 @@ public class SYMKNavigationViewController: SYMKModuleViewController {
     private var infobarController: SYMKInfobarController?
     private var speedController: SYMKSpeedController?
     private let routePreviewController = SYMKRoutePreviewController()
+    private let laneAssistController = SYMKLaneAssistController()
 
     private var instructionsController: SYMKDirectionController? = SYMKDirectionController() {
         didSet {
             guard let navigationView = view as? SYMKNavigationView, let instructionsController = instructionsController else {
                 (view as? SYMKNavigationView)?.setupInstructionView(nil)
+                (view as? SYMKNavigationView)?.setupLaneAssistView(nil)
```

Comment:
lane assist asi nemusi byt zavisly od instructions controllera ci? jedine ked mas ten full signpost controller tak vtedy lebo je integrovany v signposte vlastne nie?

Reply:
Hm, no ono laneassist je ako samostatna komponenta, cize si ju vies zobrat a dat hocikde. Ale ano v navigacnom module som to momentalne urobil tak ze bud mas signpost a on si to handluje sam, ale ked mas directions tak je to pod tym. Malo by nejaky zmysel ze si nezapnem directions a potom co? bude to upnute len tak samo o sebe hore? Prislo mi to trochu divne ake by to malo vyuzitie. Ale ak chceme, mozeme sa dohodnut na nejakej komplexnejsej viac dynamickej logike.